phil/s013-course-periods.md

```diff
@@ -13,7 +44,7 @@ Screens:
 ```coffee
```

Comment:
Kind of a tangent, but what will y'all want in here to differentiate physics vs bio (so you know which banner images to use, etc)